Mage.Common/src/main/java/mage/view/TableView.java

```diff
@@ -174,12 +176,15 @@ public TableView(Table table) {
                     if (tourneyMatchOptions.getFreeMulligans() > 0) {
                         infoText.append(" FM: ").append(tourneyMatchOptions.getFreeMulligans());
                     }
-                    if (table.getMatch().getOptions().isCustomStartLifeEnabled()) {
-                        infoText.append(" StartLife: ").append(table.getMatch().getOptions().getCustomStartLife());
-                    }
-                    if (table.getMatch().getOptions().isCustomStartHandSizeEnabled()) {
-                        infoText.append(" StartHandSize: ").append(table.getMatch().getOptions().getCustomStartHandSize());
-                    }
+                    table.getMatch().ifPresent(match -> {
+                        // TODO for review: This technically isn't possible since a match isn't a tournament?
```

Comment:
Going to check later if this can be grabbed from the tournament. Will then create a base interface of shared properties if so.